databroker/src/broker.rs

```diff
@@ -1203,7 +1203,7 @@ impl<'a, 'b> DatabaseReadAccess<'a, 'b> {
     }
 }
 
-impl<'a, 'b> DatabaseWriteAccess<'a, 'b> {
+impl DatabaseWriteAccess<'_, '_> {
```

Comment:
Why did you changed from explicit lifetimes to anonymous lifetimes? did it complain?

Reply:
This is inherited from #114, see also https://rust-lang.github.io/rust-clippy/master/index.html#needless_lifetimes

As we use "latest and greatest" clippy we get these types of regressions now and then

(long term we could discuss if we rather should have a fixed rust/clippy/fmt version and perform updates only when preparing for a (major) release, to reduce the risk of regressions)